lib/src/disposable/disposable.dart

```diff
@@ -427,6 +428,28 @@ class Disposable implements _Disposable, DisposableManagerV3 {
     return null;
   }
 
+  void subscribeToDocumentEvent(String event, EventListener callback,
+      {bool useCapture: null, EventTarget documentObject: null}) {
+    if (documentObject == null) {
+      documentObject = document;
+    }
+    _subscribeToEvent(documentObject, event, callback, useCapture);
+  }
+
+  void subscribeToDomElementEvent(
+      Element element, String event, EventListener callback,
+      {bool useCapture: null}) {
+    _subscribeToEvent(element, event, callback, useCapture);
+  }
+
+  void subscribeToWindowEvent(String event, EventListener callback,
+      {bool useCapture: null, EventTarget windowObject: null}) {
```

Comment:
Default value is already null, so you can leave those off. You might say the "default default value" is null...